### DISCUSSION TITLE
"Cannot execute code. Unable to start a runtime for the R language." in Linux

### DISCUSSION THREAD

**Discussion by ZhangMH2000** [posit-dev/positron#4498] [Q&A] [answered]
Hello Positron developers,

I'm using Ubuntu 22.04 and have downloaded Positron [2024.08.0-48]. When I try to run R scripts, I encounter the error: "Cannot execute code. Unable to start a runtime for the R language." I'm confident that R 4.3.2 is installed in my conda environment.

I checked issue #4280  and saw that version [2024.08.0-31] “addresses a problem with Positron Windows installations locating R versions”. However, the deb version of [2024.08.0-31] also produces the same error for me. I've tried some earlier versions as well, but none have worked.

Could you please provide some suggestions? Thank you!

**Comment by juliasilge** [comment, marked as answer]
We don't currently automatically detect R installed via conda and have https://github.com/posit-dev/positron/issues/4398 to track that feature.

In the meantime, I suggest you open Positron as outlined in https://github.com/posit-dev/positron/discussions/4385#discussioncomment-10386182:

- Put Positron on the path, [like this](https://github.com/posit-dev/positron/issues/1310#issuecomment-1719796378)
- Activate your conda env
- Launch Positron from the shell where your conda env is activated, like via `positron .`

**Reply by ZhangMH2000** [reply]
Thank you! In fact I always start RStudio in different conda environments so that I can run R scripts with various R versions and packages. Really thanks!
